File: crons-sample/src/main/java/com/raxdenstudios/sample/SampleCronProcedureService.java
package com.raxdenstudios.sample;

import com.raxdenstudios.cron.model.Cron;
import com.raxdenstudios.cron.service.CronProcedureService;

/**
 * Created by agomez on 26/02/2016.
 */
public class SampleCronProcedureService extends CronProcedureService {

    private static final String TAG = SampleCronProcedureService.class.getSimpleName();

    @Override
    protected void onCronLaunched(Cron cron) {

    }

}

File: crons-sample/src/main/java/com/raxdenstudios/sample/SampleCronProcedureService.kt
package com.raxdenstudios.sample

import com.raxdenstudios.cron.model.Cron
import com.raxdenstudios.cron.service.CronProcedureService

/**
 * Created by agomez on 26/02/2016.
 */
class SampleCronProcedureService : CronProcedureService() {

    companion object {
        private val TAG = SampleCronProcedureService::class.java.simpleName
    }

    override fun onCronLaunched(cron: Cron) {

    }

}
